tutorials/3d/softbody.rst

```diff
@@ -0,0 +1,64 @@
+.. _doc_softbody:
+
+Soft body
+=========
+
+Soft body simulates cloth movement and other deformable bodies.
+
+Basic set-up
+~~~~~~~~~~~
+
+A :ref:`SoftBody <class_SoftBody>` node is used for soft body simulations.
+
+We will create a bouncy cube to demonstrate the set up of a soft body.
+
+Open a new scene with a ``Spatial`` node as root. Then, create a ``Softbody`` node. Add a ``CubeMesh`` in the ``mesh`` property of the node in the inspector and increase the subdivision of the mesh for simulation.
+
+.. image:: img/softbody_cube.png
+
+Set the parameters to obtain the type of soft body you aim for. Try to keep the ``Simulation Precision`` above 5, otherwise, the soft body may collapse.
+
+.. image:: img/softbody_cube_menu.png
+
+.. note:: Handle some parameters with care, as some value can lead to strange results. For example, if the shape is not completely closed and you set pressure to more than 0.
```

Comment:
You give an example, but it's odd to me that you don't explain what happens if you try that example.